client/asset/dcr/dcr.go

```diff
@@ -2411,6 +2408,11 @@ func (dcr *ExchangeWallet) parseUTXOs(unspents []walletjson.ListUnspentResult) (
 			}
 			return nil, fmt.Errorf("error reading asset info: %w", err)
 		}
+		if nfo.ScriptType == dexdcr.ScriptUnsupported {
+			// InputInfo does this for P2SH with non-standard redeem scripts.
+			// Don't return these since they cannot fund arbitrary txns.
+			continue
+		}
```

Comment:
There's no change elsewhere that requires this, I believe we were just overlooking this.